pkg/controller/plan/adapter/converter.go

```diff
@@ -99,42 +95,62 @@ func (c *Converter) ConvertPVCs(pvcs []*v1.PersistentVolumeClaim, srcFormat srcF
 	return false, nil
 }
 
-func (c *Converter) ensureJob(pvc *v1.PersistentVolumeClaim, srcFormat, dstFormat string) (*batchv1.Job, error) {
-	jobName := getJobName(pvc, "convert")
-	job := &batchv1.Job{}
-	err := c.Destination.Client.Get(context.Background(), client.ObjectKey{Name: jobName, Namespace: pvc.Namespace}, job)
+func (c *Converter) deleteScratchDV(scratchDV *cdi.DataVolume) {
+	err := c.Destination.Client.Delete(context.Background(), scratchDV, &client.DeleteOptions{})
```

Comment:
can we drop the last argument here as well?
```suggestion
	err := c.Destination.Client.Delete(context.Background(), scratchDV)
```